app/controllers/concerns/scholars_archive/works_controller_behavior.rb

```diff
@@ -12,8 +12,32 @@ def edit
       super
     end
 
+    def update
+      set_other_option_values
```

Comment:
@revgum yeah I tried calling it with `:before_action`, but for some reason `curation_concern` is nil when the method is ran